hail/src/main/scala/is/hail/io/fs/GoogleStorageFS.scala

```diff
@@ -5,15 +5,19 @@ import java.net.URI
 import java.nio.ByteBuffer
 import java.nio.file.FileSystems
 
+import org.apache.log4j.{LogManager, Logger}
 import com.google.auth.oauth2.ServiceAccountCredentials
 import com.google.cloud.{ReadChannel, WriteChannel}
 import com.google.cloud.storage.Storage.BlobListOption
 import com.google.cloud.storage.{Blob, BlobId, BlobInfo, Storage, StorageOptions}
+import is.hail.HailContext
 
 import scala.collection.JavaConverters._
 import scala.collection.mutable
 
 object GoogleStorageFS {
+  val log = Logger.getLogger(getClass.getName())
```

Comment:
if you make this `private` its still accessible in GoogleStorageFS's instances but seems tidier.